<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbPathName">
     <property name="text">
      <string>文件名称：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edPathName"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>输出文件名称：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edOutName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnFile">
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPath">
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBackup">
       <property name="text">
        <string>备份</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestore">
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="txtLogs"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
